content/Dealers & Artists/drip/index.md

```diff
@@ -11,328 +11,336 @@ banner_author: "Blue Hasia"
 
-In the event you are unable to get a table from the waitlist, any Dealer or Dealer Assistant Membership can become an Attending Membership at the listed price. Similarly, Dealer-Sponsors can be Sponsors and Dealer-Supersponsors can be Supersponsors. However, we will not convert memberships unless the request is emailed prior to the close of Pre-registration on June 16th, 2024. These Memberships may also be canceled until the close of Pre-registration; please email [**refunds@anthrocon.org**](mailto:refunds@anthrocon.org) if a refund is needed.
+In the event you are unable to get a table from the waitlist, any Dealer or Dealer Assistant Membership can become an Attending Membership at the listed price. Similarly, Dealer-Sponsors can be Sponsors and Dealer-Supersponsors can be Supersponsors. However, we will not convert memberships unless the request is emailed prior to the close of Pre-registration on June 13th, 2025. These Memberships may also be canceled until the close of Pre-registration; please email <refunds@anthrocon.org> if a refund is needed.
 
 If you find that you will not need or cannot use any or all of your Dealership space, please inform us as soon as possible. The earlier we know, the easier it will be for a replacement Dealer to make travel plans for Anthrocon. It is against Anthrocon's policy to transfer an unwanted table to another person to bypass the waiting list.
 
-Table or Memberships cancellations should be directed to both [**dealers@anthrocon.org**](mailto:dealers@anthrocon.org) and [**refunds@anthrocon.org**](mailto:refunds@anthrocon.org). If canceling a table, please mention explicitly whether you wish to cancel your Dealer membership or have it converted to a standard Membership.
+Table or Memberships cancellations should be directed to both <dealers@anthrocon.org> and <refunds@anthrocon.org>. If canceling a table, please mention explicitly whether you wish to cancel your Dealer membership or have it converted to a standard Membership.
 
-Full refunds will be given for tables that are canceled on or before April 30th, 2024. **No refunds will be given for tables canceled after this.**
+Full refunds will be given for 2025 tables that are canceled on or before April 30th, 2025. No refunds will be given for tables canceled after this date!
 
-Full refunds will be given for 2023 memberships that are canceled on or before June 14th, 2024. **No refunds will be given for memberships canceled after this.** Dealerships can be ‘rolled over’ to the next year’s convention.
+Full refunds will be given for 2025 Memberships that are canceled on or before June 8th, 2025. No refunds will be given for memberships canceled after this date! Neither Memberships nor Dealerships can be ‘rolled over’ to the next year’s convention, and ‘vouchers’ are no longer provided.
```

Comment:
Confirm full refund or partial in line with general refund policy.

Reply:
Full refund is applicable.